packages/rest-api-client/package.json

```
@@ -15,7 +15,9 @@
   "types": "lib/src/index.d.ts",
   "scripts": {
     "prebuild": "pnpm clean",
-    "build": "tsc --build --force",
+    "build": "run-p compile:*",
+    "compile:browser": "tsc --build --force tsconfig.json",
+    "compile:node": "tsc --build --force tsconfig.node.json",
```

Because all the source files are transpiled with `tsconfig.json`, we don't need to generate output with `tsconfig.node.json`.
(The `src/**/node.ts` and `UnsupportedPlatformError.ts` are generated twice)

The `tsconfig.node.json` will be only for type checking.

```suggestion
    "build": "run-p compile typecheck",
    "compile": "tsc --build --force tsconfig.json",
    "typecheck": "tsc --build --force tsconfig.node.json",
```

In addition, we want to find browser-only dependencies not only on `node.ts` and `UnsupportedPlatformError.ts` but also all the files excluding `browser.ts` and its related files.